### PR TITLE
lynx: Update 20231231

### PR DESCRIPTION
**Pull request by dominator-prime**

### CODE REVIEW

File: lynx.json
Comment (gwolf2u) [inline lynx.json:1]: buildtype typo

File: lynx_changelog.txt
Comment (gwolf2u) [inline lynx_changelog.txt:1]: Can continue adding on top, just to have history

**Review by gwolf2u — COMMENTED** [review]
Next time, push directly
 Don't pull request and merge your own

File: lynx_changelog.txt
Comment (dominator-prime) [reply lynx_changelog.txt:1]: first time 
sure will take care all of these  next time "))

**Pull request merged**